src/components/CatalogCard/CatalogCard.tsx

```diff
@@ -141,27 +142,32 @@ export const CatalogCard: FunctionComponent<CatalogCardProps> = ({
                   LANGUAGES.indexOf(right as Language)
               )
               .map(([language, info]) => {
-                const isSupported =
+                const isSupportedByConstruct =
```

Comment:
I'd rather call this `isSupportedByLibrary` or `isSupportedByPackage`. It's otherwise too close to `isSupportedByConstructHub` and makes me question my eyes quite a bit...